code/modules/research/xenobiology/crossbreeding/warping

```diff
@@ -0,0 +1,865 @@
+	new/datum/stack_recipe("Resin nest", /obj/structure/bed/nest , 5, one_per_turf = 1, on_floor = 0, time = 50), \ 
+	))
+
+/obj/item/stack/sheet/xeno_resin/get_main_recipes()
+	.= ..()
+	. += GLOB.resin_recipes
+
+
+
+
+
+
+
+/* pink rune, makes people slightly happier when they get hugged on it */
+
+/obj/item/slimecross/warping/pink
+	colour = "pink"
+	effect_desc = "Draws a rune that gives makes everyone that walks on it happier!"
+	runepath = /obj/effect/warped_rune/pinkspace
+
+
+
+/obj/effect/warped_rune/pinkspace
+	desc = "Love is the only reliable source of happiness we have left. But like everything, it comes with a price."
+	icon_state = "love_rune"
+
+
+///adds the hug mood effect along with the sound effect.
+/obj/effect/warped_rune/pinkspace/Crossed(atom/movable/AM)
+	if(istype(AM,/mob/living/carbon/human))
+		playsound(T, "sound/weapons/thudswoosh.ogg", 50, TRUE)
+		SEND_SIGNAL(AM, COMSIG_ADD_MOOD_EVENT,"jolly", /datum/mood_event/jolly)
+		AM.visible_message(AM, "<span class='notice'>You start feeling happier somehow.</span>")
+
+
+
+
+/*Gold rune : Turn things over it to gold, completely fucks over economy */
+
+/obj/item/slimecross/warping/gold
+	colour = "gold"
+	runepath = /obj/effect/warped_rune/goldspace
+	effect_desc = "Draws a rune capable of turning nearly anything put on it to gold."
+
+/obj/effect/warped_rune/goldspace
+	icon_state = "midas_rune"
+	desc = "When everything is abundant, it becomes easy to forget when one had to work to obtain anything."
+
+/obj/effect/warped_rune/goldspace/Initialize()
+	.=..()
+	RegisterSignal(get_turf(src),COMSIG_ATOM_ENTERED,.proc/midas_touch)
+
+///turn anything on the tile to a golden version of themselves. Sheets of any kind will be directly turned into gold bars.
+/obj/effect/warped_rune/goldspace/proc/midas_touch()
+	for(var/obj/item/stack/sheet/S in T)
+		if(istype(S,/obj/item/stack/sheet/mineral/gold)) //can't turn gold into gold
+			return
+		var/obj/item/stack/sheet/mineral/gold/G = new(T)
+		G.amount = S.amount //as broken as it sounds if you mass produce easy stacks
+		qdel(S)
+		return
+
+	for(var/obj/item/I in T)
+		I.material_flags = MATERIAL_COLOR | MATERIAL_ADD_PREFIX | MATERIAL_AFFECT_STATISTICS
+		I.set_custom_materials(list(/datum/material/gold=500)) //always set at 500 to not make it too abusable
+
+
+/*Adamantine rune, will spawn ores depending on the mineral rocks surrounding it. Here to make miners do their job even less.  */
+
+/obj/item/slimecross/warping/adamantine
+	colour = "adamantine"
+	runepath = /obj/effect/warped_rune/adamantinespace
+	effect_desc = "draws a rune capable of copying the ores of nearby mineral rocks."
+
+/obj/effect/warped_rune/adamantinespace  //doesn't have a rune icon yet please spriters help me I can't sprite for shit I beg you
+	desc = "The universe's ressource are nothing but tools for us to use and abuse."
+	max_cooldown = 300  //"mines" things every 30 seconds.
```

Comment:
generally, this is a pretty shitty rune. by the time you get this, the miners should have already mined all of the ores you'll ever need. if they haven't, just grabbing a pickaxe, ore bag, and mining scanner from the public part of the mining base and mining ore manually is going to be less effort than making and setting one of these things up and then going back and harvesting a mere 20 chunks of ore from it 10 minutes later, hell, instead of making this crossbreed from an adamantine slime, you could just make a servant golem shell from it and tell your new servant golem to go mine mats for you.

Reply:
tldr;
this rune is way too slow, requires too much effort/costs too much to make for what it does, and the thing that it's supposed to do for you should have been done an HOUR ago by your miners.

Reply:
The main point of this rune is to not only act as free mining point, but also to replace miners if they somehow didn't do their job and potentially getting a lot of the rarer ores. While it can't per say replace the speed of actual miners it does something that doesn't require any effort apart from just placing it and getting it later.